lib/src/native_updater.dart

```diff
@@ -7,9 +7,15 @@ import 'updgrade_material_alert.dart';
 import 'upgrade_cupertino_alert.dart';
 
 class NativeUpdater {
+  bool _forceUpdate;
   String _appName;
   String _packageName;
   String _appStoreUrl;
+  String _titlePrefix;
+  String _description;
+  String _updateButtonLabel;
+  String _closeButtonLabel;
+  String _ignoreButtonLabel;
 
   /// Singleton
   static final NativeUpdater _singleton = NativeUpdater._internal();
```

Comment:
`_singleton` is a bad name.

I suggest you change to:

_nativeUpdater
_nativeUpdaterInstance

Or anything that gives more context of what the variable is.

Reply:
Will change to _nativeUpdaterInstance

Reply:
Resolved in #5